test/e2e/node/pod_resize.go

```diff
@@ -1408,14 +1441,133 @@ func doPodResizeErrorTests() {
 	}
 }
 
-var _ = SIGDescribe("Pod InPlace Resize Container [Feature:InPlacePodVerticalScaling]", func() {
-	featureGatePostAlpha = true
-	if fs, found := utilfeature.DefaultFeatureGate.DeepCopy().GetAll()[features.InPlacePodVerticalScaling]; found {
-		if fs.PreRelease == featuregate.Alpha {
-			featureGatePostAlpha = false
+func doPodResizeSchedulerTests() {
+	f := framework.NewDefaultFramework("pod-resize-scheduler")
+
+	ginkgo.It("pod-resize-scheduler-testcase1", func() {
+		nodes, _ := f.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		nodeNum := len(nodes.Items)
+		if nodeNum > 1 {
+			framework.Logf("There are more than 1 node in the testing cluster, so skip the pod-in-queue-resize-test!")
```

Comment:
Instead of skipping the test (false positive), we should fail the test. 
But maybe we can update the pod spec to force scheduler to choose one node using node affinity in the pod spec.
For e.g. in YAML I force my pod on a particular node in a KinD cluster as follows:
```yaml
spec:
  affinity:
    nodeAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        nodeSelectorTerms:
        - matchExpressions:
          - key: kubernetes.io/hostname
            operator: In
            values:
            - kind-worker
```
This can be done programmatically and set to node.Name?